code/datums/outfit.dm

```diff
@@ -208,6 +203,12 @@
 		else
 			WARNING("Unable to equip accessory [accessory] in outfit [name]. No uniform present!")
 
+	for(var/obj/item/equipped_item in H.contents) //before suit storage, as equipped items for example could extend a suit item
```

Comment:
I don't like this exception and calling utility function on only part of the outfit here. What's the special case this covers ?

Reply:
modsuits are back mounted, so they need to extend their chestplate before theres a suit storage item put on them, and i wouldnt count suit storage items as really clothing just like pockets